### PR TITLE
Create link to root page with dxw logo

### PR DESCRIPTION
**Pull request by LornaHarwood**
Created a link using a dxw branded image which directs to the root page.
This sits within the Nav bar.

### CODE REVIEW

**Comment by LornaHarwood** [comment]
[Home button on Navbar](https://trello.com/c/XUqLUEDP/28-home-button-on-navbar)

File: app/assets/stylesheets/pages.scss [hunk 1]
```diff
@@ -47,7 +47,6 @@ main {
   position: fixed;
   top: 0;
   left: 0;
-  padding: 10px;
```
Comment (erbridge) [inline app/assets/stylesheets/pages.scss:50]: Do we want to remove this padding?

File: spec/features/user_can_navigate_back_to_home_page_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.feature "User can navigate straight back to root page from any location" do
+  context "When user is on any page" do
+    scenario "the dxw image will direct user to the pages root page" do
```
Comment (erbridge) [inline spec/features/user_can_navigate_back_to_home_page_spec.rb:5]: This scenario name assumes things about the implementation. What happens with this test if I replace the image with some text saying "Home"?

Comment (erbridge) [reply spec/features/user_can_navigate_back_to_home_page_spec.rb:5]: Alternatively, how can we make the test better reflect the wording of the scenario?

**Pull request merged**